bigquery/main.go

```diff
@@ -134,23 +167,105 @@ func (n *bqNotifier) SendNotification(ctx context.Context, build *cbpb.Build) er
 		log.Warningf("Build passes filter but does not have trigger ID: %v, status: %v", n.filter, build.GetStatus())
 	}
 	log.Infof("sending Big Query write for build %q (status: %q)", build.Id, build.Status)
+	if build.ProjectId == "" {
+		return fmt.Errorf("Build missing project id")
+	}
+	if build.Id == "" {
+		return fmt.Errorf("Build missing id")
+	}
+	newRow := bqRow{}
+	newRow.ProjectID = build.ProjectId
+	newRow.ID = build.Id
+	newRow.BuildTriggerID = build.BuildTriggerId
+	newRow.Status = build.Status.String()
+	if newRow.Status == "STATUS_UNKNOWN" {
+		return fmt.Errorf("Build %v missing status", newRow.ID)
+	}
+	terminalBuild := false
+	for _, status := range terminalStatusCodes {
+		if status == newRow.Status {
+			terminalBuild = true
+		}
+	}
+	if !terminalBuild {
+		log.Infof("Not writing non-terminal build step %v", newRow.Status)
+		return nil
+	}
+
+	log.Infof("BUILD IMAGES %v", build.GetImages())
+	newRow.Images = []*buildImage{}
+	for _, image := range build.GetImages() {
+		buildImage, err := parseImageManifest(image)
+		if err != nil {
+			return fmt.Errorf("Error parsing image manifest: %v", err)
+		}
+		log.Info(buildImage)
```

Comment:
What is this log statement for? Can you make this log statement more descriptive?